### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CStewCodes**
Glitzy new stuff if I do say so myself... GO, Check it out, don't read it all as code, go look at it as a preview!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,14 +13,18 @@
...
+This project is the culmination of months of **🚀 hard work 🚀** growing our skills within the Galvanize course. We came together as a team to rework an existing project from a previous cohort in order to meet the demands of a client.
 
-- Project Manager: Chris Stewart (pri), Rane Gray (alt)
-- UI Manager: Rane Gray (pri), Daniel Purdy (alt)
-- Architecture Manager: Joe Goralczyk (pri), Jordan Carlson (alt)
+-  👨🏻‍✈️ 📝 Project Manager: Chris Stewart (pri), Rane Gray (alt)
+-  👨🏻‍🎨 🖼️ UI Manager: Rane Gray (pri), Daniel Purdy (alt)
+-  👷🏻‍♂️ 🏗️ Architecture Manager: Joe Goralczyk (pri), Jordan Carlson (alt)
 
 Working together as an Agile group, we worked hard to redesign this application visually and functionally per our client's needs. We've reworked the authentication system that correctly stores hashed passwords in the database and keeps the user logged in, a dashboard with an interactive table element that allows for tracking prospective stuents, and an interview page that implements a usable coding space to collaborate between admissions personnel and students.
```
Comment (jordan-carlson99) [inline README.md:29]: I think we should include that we use web tokens, it seemed like a real achievement to get them working. Maybe "we refactored authentication to add protected routes and json web tokens along with hashed password storage" and with the shared coding space include we use websockets "a usable coding space to collaborate between admissions personnel and students using an express websocket server and sandpack code editors". Adding those technical details to give a little more weight to the amount of understanding that stuff requires.

Comment (jordan-carlson99) [inline README.md:29]: also "stuents" on line 2

File: README.md [hunk 2]
```diff
@@ -30,38 +34,70 @@ Working together as an Agile group, we worked hard to redesign this application
...
+  [![React-Auth](https://img.shields.io/badge/-React_Auth-red?style=flat&logo=npm&logoColor=black)](https://www.npmjs.com/package/react-auth-kit)
+  [![Jest](https://img.shields.io/badge/-Jest-purple?style=flat&logo=jest&logoColor=black)](https://jestjs.io/)
+  [![Bootstrap](https://img.shields.io/badge/-Bootstrap-7952B3?style=flat&logo=bootstrap&logoColor=white)](https://getbootstrap.com/)
+  [![Vite](https://img.shields.io/badge/-Vite-646CFF?style=flat&logo=vite&logoColor=F6DC40)](https://vitejs.dev/)
+  
+  [![Node.js](https://img.shields.io/badge/-Node.js-339933?style=flat&logo=Node.js&logoColor=black)](https://nodejs.org/)
+  [![Express.js](https://img.shields.io/badge/-Express.js-000000?style=flat&logo=express&logoColor=white)](https://expressjs.com/)
+  [![Supabase](https://img.shields.io/badge/-Supabase-3FCF8E?style=flat&logo=supabase&logoColor=black)](https://supabase.com/)
+  [![Code Sandbox](https://img.shields.io/badge/Code%20Sandbox-Live%20code%20editor-blue?style=plastic&logo=codesandbox)](https://codesandbox.io/?from-app=1)
+
```
Comment (jordan-carlson99) [inline README.md:58]: i think socket.io is a worth mention if that has a badge

**Review by jordan-carlson99 — COMMENTED** [review]
looks legit man i like it, i think if we focus on system design like rane was saying we could add some sort of architecture section at the end of the next sprint.  It might also be cool to add some before/after pictures for the ui once its polished. Just spit-balling here

File: README.md (hunk 1 above)
Comment (joegor85) [reply README.md:29]: Wow, I'm so sorry. I'm usually nitpicky about typos like that, lol. Good catch

**Comment by joegor85** [comment]
Nice work, Chris. I think just make those small adjustments that Jordan mentioned and it will be awesome.

File: README.md (hunk 2 above)
Comment (CStewCodes) [reply README.md:58]: Ohh, thank you thats right I forgot thank you

**Comment by joegor85** [comment]
In addition to the "stuents" mispelling, I just caught another one that needs to be fixed. At the very very bottom of the page when you link to our linkedin profiles, my last name is just missing a "k" at the end. Should be  "Goralczyk".

File: README.md (hunk 2 above)
Comment (CStewCodes) [reply README.md:63]: Done

File: README.md (hunk 1 above)
Comment (CStewCodes) [reply README.md]: Done

Comment (CStewCodes) [reply README.md]: Done

**Pull request merged**